src/settings/defaults.py

```diff
@@ -372,8 +369,8 @@ def __setattr__(self, k, v):
 
     d.LDAP_ENABLED = False
     d.AUTHENTICATION_BACKENDS = (
-	'accounts.ldap_auth.LDAPBackend',
-	'django.contrib.auth.backends.ModelBackend',
+	    'accounts.ldap_auth.LDAPBackend',
+	    d.AUTH_BACKEND,
     )
     d.LDAP_URI="ldap://ldap.DAMANENAME.TOPLEVEL" 
     d.LDAP_BASE="dc=DAMANENAME,dc=TOPLEVEL"
```

Comment:
Your changes are correct from my point of view.

Oh, found small typography error in "my" code: 
`d.LDAP_BASE="dc=DAMANENAME,dc=TOPLEVEL"`
should be
`d.LDAP_BASE="dc=DOMAINNAME,dc=TOPLEVEL"`
:-)

Reply:
Alright, while we're at it... I fixed it :)